src/components/breadcrumb/index.njk

```diff
@@ -10,9 +10,8 @@
 {# componentNunjucks #}
 {# componentHtml #}
 
-{% block componentGuidanceLink %}
-  http://www.linktodesignsystem.com
-{% endblock %}
+{# override link to design system here if it's diffrent to base url + componentName #}
```

Comment:
Tiny thing, there's a typo - different, needs updating for all comments here. Apart from that, super!